src/consensus/consensus.h

```diff
@@ -1,16 +0,0 @@
-// Copyright (c) 2009-2010 Satoshi Nakamoto
```

Comment:
Damm, consensus.h is going to be destroyed before I was able to move a single function declaration...Sniff sniff